```py
ValueError: You did something wrong which is bad.

```